docs/dev/developer-guides/hello-world.md

```diff
@@ -503,6 +503,15 @@ async changeGreeting() {
 
 You now have a fully functional Greeter-dApp! However, it does not leverage any zkSync-specific features.
 
+::: warning
+
+What happens when you get a **wallet_requestPermissions** error?
+
+To fix this error, refresh your browser, another fix is opening the MetaMask extension on your browser and click *Next* or *Cancel* to resolve it.
```

Comment:
To fix this error, refresh your browser, or open the MetaMask extension on your browser and click *Next* or *Cancel* to resolve it.